data/tests/fwupdmgr.sh

```diff
@@ -268,5 +268,10 @@ echo "Getting approved firmware..."
 fwupdmgr get-approved-firmware
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Run security tests..."
+fwupdmgr security
+rc=$?; if [ $rc = 1 ]; then error $rc; fi
+
```

Comment:
run it both normal and with `--json`.  Which reminds me; maybe we need something that checks json is valid like `jsonlint`?

Reply:
I don't think json-glib is super likely to output invalid json; if we screw up the builder we're going to get critical warnings not output.

Reply:
I haven't dug into it yet; but https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1095450

Reply:
And also; we should still call it twice with / without `--json` so we have coverage on the few lines different in `fu-util.c`

Reply:
Added the --json